packages/react-components/src/trace-explorer/trace-explorer-opened-traces-widget.tsx

```diff
@@ -63,9 +60,9 @@ export class ReactOpenTracesWidget extends React.Component<ReactOpenTracesWidget
 
     componentWillUnmount(): void {
         signalManager().off(Signals.EXPERIMENT_OPENED, this._onExperimentOpened);
-        signalManager().off(Signals.EXPERIMENT_CLOSED, this._onExperimentClosed);
+        signalManager().off(Signals.EXPERIMENT_DELETED, this._onExperimentDeleted);
         signalManager().off(Signals.TRACEVIEWERTAB_ACTIVATED, this._onOpenedTracesWidgetActivated);
-        signalManager().off(Signals.OPENED_TRACES_UPDATED, this._onOpenedTracesUpdated);
+        signalManager().off(Signals.OPENED_TRACES_UPDATED, this._onExperimentClosed);
```

Comment:
Should be Signals.EXPERIMENT_CLOSED.

Also I suggest to group the 3 experiment signals together, just because it's nicer. ;)